Packs/MISP/Integrations/MISPV3/MISPV3.py

```diff
@@ -1016,9 +1027,29 @@ def search_attributes(demisto_args: dict) -> CommandResults:
     pagination_args_validation(page, limit)
 
     response = PYMISP.search(**args)
+
     if response:
         response_for_context_list = []
         response_for_context_dict = {}
+
+        if 'with_attachments' in args and args.get('with_attachments') == 1:
+            if isinstance(response, list):
+                for attachment in response:
+                    if 'Event' in attachment:
+                        if 'Object' in attachment.get('Event'):
+                            for objects in attachment.get('Event').get('Object'):
```

Comment:
```suggestion
                    for objects in attachment.get('Event', {}).get('Object', []):
```
We don't have to check if the key exist we can return a default value if the key doesn't exist.